README.md

```diff
@@ -212,6 +212,13 @@ The remote data are originally stored at the remote web-site of one of the
 matrix collections. Before they are presented to the user, they are downloaded
 to local disk storage, which serves as a permanent cache.
 
+By default, the data directory is a scratchspace managed by [`Scratch.jl`](https://github.com/JuliaPackaging/Scratch.jl), but can be changed by setting the `MATRIXDEPOT_DATA` environment variable.
+
+The data directory can be queried by
+
+    julia> MatrixDepot.data_dir()
+    "/home/.../.julia/scratchspaces/b51810bb-c9f3-55da-ae3c-350fc1fbce05/data
+
```

Comment:
this text, which belongs to #75, is already in the master branch via #81.

Reply:
Good catch! fixed in https://github.com/JuliaMatrices/MatrixDepot.jl/pull/79/commits/389d9ba61a730a2c8cca656e5108857896ffda16